charts/anchore-policy-validator/templates/whitelist-crd.yaml

```diff
@@ -1,35 +1,42 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: whitelistitems.security.banzaicloud.com
 spec:
   group: security.banzaicloud.com
-  version: v1alpha1
   names:
     kind: WhiteListItem
     plural: whitelistitems
     singular: whitelistitem
+    shortNames:
+      - wl
```

Comment:
**Question**: why has the `shortNames` been relocated from `deploy/whitelist-crd.yaml` to `charts/anchore-policy-validator/templates/whitelist-crd.yaml`?

Reply:
The deploy dir is used for testing purposes, while charts dir contains the hem chart. So the shortNames field was missing before in the case of the helm chart.